packages/odie-client/src/utils/zendesk-message-converter.ts

```diff
@@ -11,6 +11,13 @@ function convertUrlsToMarkdown( text: string ): string {
 
 	return text.replace( urlRegex, ( match ) => {
 		let url = match;
+
+		// Remove trailing punctuation if it's only a sentence-ending character
+		const trailing = url.match( /[.,!?;:]+$/ );
+		if ( trailing ) {
+			url = url.slice( 0, -trailing[ 0 ].length );
+		}
```

Comment:
This seems to be the equivalent of 

```suggestion
	url = url.replace( /[.,!?;:]+$/g, '');
```